hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -1502,12 +1502,11 @@
 </property>
 
 <property>
-    <name>fs.s3a.metadatastore.authoritative.dir.ttl</name>
-    <value>3600000</value>
+    <name>fs.s3a.metadatastore.metadata.ttl</name>
+    <value>15m</value>
     <description>
-        This value sets how long a directory listing in the MS is considered as
-        authoritative. The value is in milliseconds.
-        MetadataStore should be authoritative to use this configuration knob.
+        This value sets how long a metadata in the MS is valid. The value is in
```

Comment:
text only makes sense for us developers. 

"This value sets how long an entry in the MetadataStore is considered valid. After this time has expired, in non-authoritative mode, S3Guard will check the S3 store for the existence/state of an object and use that as its source of truth.
This is needed when other applications are writing/updating
data in the samme store without going through S3Guard."